src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,51 @@
+package homework_2.random_chars_table;
+
+import homework_2.bufferReaders.IOHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RandomCharsTable {
```

Comment:
Seems there are problems with even/odd segregation

Reply:
Look like works correctly, can you share more details ?

Reply:
Right, looks like it